doc-content/enterprise-only/getting-started/test-speed-proc.adoc

```diff
@@ -0,0 +1,49 @@
+
+.Prerequisites
+
+* Created the `Driver_department_traffic_violations` project
+* Created the `Violation` and `Driver` data objects
+* Created the speed violation guided decision table
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver_department_traffic_violations*.
+. Click *Add Asset* -> *Test Scenario*.
+. In the *Create new Test Scenario window* wizard, enter the following values:
+.. *Test Scenario*: `Speed limit 10-20`.
+.. *Package*: select `com.myspace.driver_department_traffic_violations`.
+. Click *Ok*.
+. Click *+GIVEN* to open the *New input* window.
+. Select *Violation* from the *Insert a new fact* pull-down menu.
+. Enter `violation` in the *Fact name* field and click *Add*.
+. Click *Add a field* located under *Insert 'Violation'[violation]* to open the *Choose a field to add* window.
+. Select *speedLimit* from the *Choose a field to add* pull-down menu and click *OK*.
+. Click (image:getting-started/6191.png[]), then click *Literal value* next to *speedLimit*.
+. Click *Literal value* in the *Field value* window, then enter `40` in the *speedLimit* field.
+. Click *Insert 'Violation'[violation]* and select *type* from the *Choose a field to add* pull-down menu in the *Choose a field to add* window, and click *OK*.
+. Click (image:getting-started/6191.png[]), then click *Literal value* next to *Literal value*.
+. Enter `Speed` in the *type* field.
+. Click *Insert 'Violation'[violation]*, and select *actualSpeed* from the *Choose a field to add* pull-down menu, and click *OK*.
+. Click (image:getting-started/6191.png[]), then click *Literal value* next to *Literal value*.
+. Enter `55` in the *actualSpeed* field.
+. Click *+Expect* to open the *New expectation* window.
+. Expand the *Rule* pull-down menu, select *Row 1 SpeedViolationRule*, and click *OK*.
+. Click *+GIVEN* to open the *New input* window, enter `trafficViolation` in the *Activate rule flow group* field, and click *Add*.
+. Click *+Expect* to open the *New expectation* window and click *Add* next to *Fact value: violation*.
+. Click *Violation 'violation' has values:* to open the *Choose a field to add* window.
+. Select *fineAmount* from the *Choose a field to add* pull-down menu and click *OK*.
+. Enter `100.0` in the *fineAmount: equals* field.
+. Click *Violation 'violation' has values:* to open the *Choose a field to add* window.
+. Select *points* from the *Choose a field to add* pull-down menu and click *OK*.
+. Enter `1` in the *points: equals* field.
+. Click *Save*, then click *Save* to confirm your changes.
+. Click *Run scenario*.
++
+
+.Speed test results screen
+image::getting-started/speedtest_results.png[Speed test results screen]
+
+The Reporting section at the bottom of the window should show a Success message. This is the expected result as the values and conditions set in the test scenario meet the requirements as specified in speed violation guided decision table.
```

Comment:
I would rephrase to lose 'should'. Something like:
If the values and conditions set in the test scenario meet the requirements as specified in speed violation guided decision table, the Reporting section at the bottom of the window displays a Success message.

Reply:
Fixed same for all test three test scenario topics.